api/service/block.ts

```diff
@@ -0,0 +1,112 @@
+import { Block } from 'shared/model/block';
+import { BlockRepo } from 'api/repository/block';
+import Response from 'shared/model/response';
+const OrderBy = require('../../helpers/orderBy.js');
+
+interface ILoadBlockParams {
+    limit: number;
+    lastId: string;
+}
+
+interface IListParams {
+    generatorPublicKey?: string;
+    numberOfTransactions?: number;
+    previousBlock?: string;
+    height?: number;
+    totalAmount?: number;
+    totalFee?: number;
+    reward?: number;
+    limit?: number;
+    offset?: number;
+    orderBy?: string;
+}
+
+export class BlockService {
+    private blockRepo = new BlockRepo();
+    private lastBlock: Block;
+    private sortFields = [
+        'id',
+        'timestamp',
+        'height'
+    ];
+
+    public loadBlocksData(params: ILoadBlockParams) : Block[] {
+        return [];
+    }
+
+    public getLastBlock(): Block {
+        // load block from db?
```

Comment:
Last block must be stored in memory

Reply:
Ok, but the thing is API doesn't know about last block changing. So currently nothing sets last block to memory, so nothing is returned by this function.